jycouet/kitql
# ✨: NEW: (EARLY) kit-feedback initialization

## 🚧 Early stage module 🚧

 ⚠️ TESTS NOT IMPLEMENTED YET ⚠️

As discussed, does not use Bob (atm).

You can : 
- create issue (with label set in config)
- see milestones filtered by string filter (ex. : [FEEDBACK]) (to set in config)
- see issues (inside milestones) filtered by label permission (set label in config to tell what user can see)
- tabs to see open / closed issues
- add comment
- show only comment with 👀 reaction (or other reaction)
- up/down vote comment AND issue (done but hidden atm)

NEXT : 

- [ ] Config refactoring
- [ ] Screenshots upload (using https://github.com/upload/policies/assets endpoint ? To test)
- [ ] Add extra (custom) metadata (ex. : page on which the user writes the issue)
- [ ] Support for comment EDIT ( & DELETE ? )
